CMakeLists.txt

```diff
@@ -9,35 +9,32 @@ project(
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/")
 
+include(GNUInstallDirs)
+
+option(ENABLE_TESTS "Build pfunit tests?" OFF)
+
 if(NOT CMAKE_BUILD_TYPE MATCHES "^(Debug|Release|RelWithDebInfo|MinSizeRel)$")
   message(STATUS "Setting build type to 'Release' as none was specified.")
-  set(CMAKE_BUILD_TYPE
-      "Release"
-      CACHE STRING "Choose the type of build." FORCE)
-  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release"
-                                               "MinSizeRel" "RelWithDebInfo")
+  set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the type of build." FORCE)
+  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif()
 
 if(NOT CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel|GNU|Clang|AppleClang)$")
-  message(
-    WARNING "Compiler not officially supported: ${CMAKE_Fortran_COMPILER_ID}")
+  message(WARNING "Compiler not officially supported: ${CMAKE_Fortran_COMPILER_ID}")
 endif()
 
 set(CMAKE_FIND_PACKAGE_PREFER_CONFIG true)
```

Comment:
Don't need this line.

Reply:
Get rid of the warning all together?

Reply:
No. The line 
```
set(CMAKE_FIND_PACKAGE_PREFER_CONFIG true)
```

Reply:
This actually is needed because we have `cmake/Modules/Find*.cmake`, so it uses the module mode, doesn't find them, then fails, and never gets to config mode.

Reply:
if we are building the package with cmake and providing the package config, why do we want to rely on hand-written Find*.cmake files.  As soon as all these libraries are transitioned  to cmake, the Find*.cmake for these libraries should be tossed.

Reply:
So, what do you think I should do about it in here? Get rid of the CMakeModules submodule and rely only on CMake configs from here on out?

Reply:
You can't yet.  You are still needing FindNetCDF.cmake.  You can create a branch in CMakeModules that progressively removes the Find*.cmake for NCEPLIBS e.g. FindBACIO.cmake, FindSIGIO.cmake etc. for the packages that we now have a package config file.  Soon the CMakeModules will be reduced to the ones not from NCEPLIBS e.g. NetCDF, ESMF, GEMPAK etc.

Reply:
Not to mention, the Find*.cmake in CMakeModules are obsolete now that the version number is removed from the library name.  So even if they are invoked, they won't work.  Another reason to ditch the hand-written ones.

Reply:
That's true